install_config/topics/ocp_osp_sdn.adoc

```diff
@@ -0,0 +1,38 @@
+[[osp-sdns]]
+=== {product-title} SDN
+
+The default {product-title} SDN is
+xref:../architecture/networking/sdn.adoc#architecture-additional-concepts-sdn[OpenShiftSDN].
+There is another option: use xref:../install_config/configure_kuryrsdn.adoc#kuryr-sdn-and-openshift[Kuryr SDN].
+
+
+[[kuryr-sdn]]
+=== Kuryr SDN
+
+Kuryr is a CNI plug-in that uses Neutron and Octavia to provide networking for pods
+and services. It is primarily designed for {product-title} clusters that run on
+OpenStack virtual machines. Kuryr improves the network performance by plugging
+{product-title} pods into OpenStack SDN. In addition it provides
+interconnectivity between {product-title} pods and OpenStack virtual instances.
+
+Kuryr is recommended for {product-title} deployments on encapsulated OpenStack
+tenant networks in order to avoid double encapsulation, such as running an
+encapsulated OpenShift SDN over an OpenStack network. Kuryr is recommended
+whenever VXLAN, GRE, or GENEVE are required
```

Comment:
In case you re-run, there's a tiny typo. Missing a dot in the end of the sentence.